docs/dev/create_a_tile.md

```diff
@@ -0,0 +1,266 @@
+| Color Profile      | sRGB                                                                                                                                              |
+| Description        | The maximum number of characters allowed is 300.                                                                                                  |
+
+Follow this template to define the `media` object in the `manifest.json` file which includes an image, a video thumbnail, and a video:
+
+{{< code-block lang="json" filename="manifest.json" collapsible="true" >}}
+"media": [
+      {
+        "media_type": "image",
+        "caption": "A Datadog Marketplace Integration OOTB Dashboard",
+        "image_url": "images/integration_name_image_name.png"
+      },
+      {
+        "media_type": "video",
+        "caption": "A Datadog Marketplace Integration Overview Video",
+        "image_url": "images/integration_name_video_thumbnail.png",
+        "vimeo_id": 123456789
+      },
+    ],
+{{< /code-block >}}
+
+For more information, see [Integrations Assets Reference][22].
+
+Before you open a pull request, run the following command to catch any problems with your integration:
+```
+ddev validate all <INTEGRATION_NAME>
+```
+
+
+## Open a pull request
+
+Push up your feature branch and open a pull request that contains your integration tile's asset files (including images) in the [`marketplace`][18] or [`integrations-extras`][26] repository. After you've created your pull request, automatic checks will run to verify that your pull request is in good shape and contains all the required content to be updated.
+
+## Review process
+
+Once your pull request passes all checks, reviewers from the `Datadog/agent-integrations`, `Datadog/marketplace-review`, and `Datadog/documentation` teams provide suggestions and feedback on best practices.
+
+Once you have addressed the feedback and re-requested reviews, these reviewers approve your pull request. Contact the Marketplace team if you would like to preview the tile in your sandbox account. This allows you to validate and preview additional changes on the tile before your pull request is merged.
+
+### How to resolve common validation errors 
+
+Out-of-the-box integrations in the `integrations-extras` repository can run into validation errors when the forked repository is out of date with the origin. Follow the steps below to resolve the validation errors by rebasing. 
+
+Updating the forked repository via the Web App
+
+1. Go to github.com
+2. Go to your repositories
+3. Select your forked repo of integrations-extras
+4. Go to "sync fork" in the top right corner 
+5. Click "update branch" 
+
+To rebase and push changes:
+
+1. `git checkout <your working branch>`
+
+2. `git rebase master`
+
+-  If there are any merge conflicts, you'd resolve them here
+
+3. `git push origin <working branch> -f`
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
```

Comment:
Just curious if this is this the end. Are we recommending them to read anything else like the GTM support for Marketplace?

Reply:
@anawishnoff I think we should include GTM support in the build an offering then link there from here

Reply:
(header 3) GTM opportunities

If you would like to publish this on the Datadog Marketplace, check out our Go-To-Market opportunities here. <link to gtm section>